pandas/core/window/rolling.py

```diff
@@ -147,12 +148,10 @@ def _validate_get_window_bounds_signature(window: BaseIndexer) -> None:
                 f"get_window_bounds"
             )
 
-    def _create_blocks(self):
+    def _create_blocks(self, obj):
```

Comment:
if you can type: Union[Series, DataFrame] (I think we have an annoation for that).